src/python/pants/backend/python/rules/coverage.py

```diff
@@ -127,20 +144,39 @@ class CoverageConfig:
     digest: Digest
 
 
+def _validate_and_update_config(coverage_config: configparser.ConfigParser) -> None:
+    if not coverage_config.has_section("run"):
+        coverage_config.add_section("run")
+        coverage_config.set("run", "branch", "True")
```

Comment:
Do you think we still want this turned on by us? I personally don't like when Pants makes decisions like this - who are we to say that you want branch data?

Reply:
I am just trying to preserve existing behavior....

Reply:
Acknowledged. To clarify, I'm proposing breaking with existing behavior.

Reply:
this kind of change should be part of its own PR.... especially since this change breaks existing integration tests.

Reply:
https://github.com/pantsbuild/pants/pull/10386